ui/packages/app/src/config.js

```diff
@@ -43,7 +43,13 @@ const config = {
       ? getEnv("REACT_APP_ALLOW_CUSTOM_TEAM")
       : true,
   PROJECT_INFO_UPDATE_ENABLED:
-    getEnv("REACT_APP_PROJECT_INFO_UPDATE_ENABLED") || false
+    getEnv("REACT_APP_PROJECT_INFO_UPDATE_ENABLED") || false,
+  LABELS_BLACKLIST: (getEnv("REACT_APP_LABELS_BLACKLIST") || "")
+    .split(",")
+    .reduce((check, label) => {
+      check[label.trim()] = true;
+      return check;
+    }, {})
```

Comment:
This looks like it may not be working as expected, could you check this once more? Thanks!
<img width="564" alt="Screenshot 2024-06-19 at 12 26 28" src="https://github.com/caraml-dev/mlp/assets/36802364/f2ee7548-06e1-4319-b3c6-13de5f142d9f">